about.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" type="text/css" media="screen and (max-width: 768px)" href=" ./aboutmobile.css">
+  <link rel="stylesheet" type="text/css" media="screen and (min-width: 768px)" href="./style.css" />
+  <script src="https://kit.fontawesome.com/686039ca8a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header class="top-header">
+    <div class="navbar-top">
+      <ul class="top-nav-ul">
+        <li><a href="#"><i class="fa-brands fa-facebook-f colorWhite"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter colorWhite"></i></a></li>
+        <li><a class="colorWhite" href="#">English</a></li>
+        <li><a class="colorWhite" href="#">My Page</a></li>
+        <li><a class="colorWhite" href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="second-navbar-top">
```

Comment:
- [ ] For semantic purposes, I would like to advise that you use the `nav` tag instead of `div` tag.